site/content/docs/03-template-syntax.md

```diff
@@ -1008,7 +1008,7 @@ The `transition:` directive indicates a *bidirectional* transition, which means
 
 Like actions, transitions can have parameters.
 
-(The double `{{curlies}}` aren't a special syntax; this is an object literal inside an expression tag.)
+(The double `{{curlies}}` isn't a special syntax; this is an object literal inside an expression tag.)
```

Comment:
The verb should agree with the plural curlies, not the singular syntax. This should stay.

Reply:
```suggestion
(The double `{{curlies}}` aren't a special syntax; this is an object literal inside an expression tag.)
```